### PR TITLE
Cut down DuplicateMapFeatureCheck run time

### PR DESCRIPTION
**Pull request by xiaohongtang2**
### Description:
Cut down DuplicateMapFeatureCheck run time
Add new lines in md file for display

Below are the run time for countries run on my machine :
1) Runs using IntegrityCheckSparkJob with one thread:
ISR which has 45 atlas files: 1.5 mins
EGY which has 91 atlas files: 17 mins
NZL which has162 atlas files:  1 hr 44 mins

2) Runs using ShardedIntegrityChecksSparkJob with one thread:
BRA which has 780 atlas files: 6 hrs 54 mins, the longest shard taking 6.4 min and most being under a minute

### CODE REVIEW

File: docs/checks/duplicateMapFeatureCheck.md [hunk 1]
```diff
@@ -2,20 +2,20 @@
 
 #### Description
 The DuplicateMapFeaturecheck flags node, way or relation which have duplicate map features in areas or connected locations.
-[Osmose#4080](https://wiki.openstreetmap.org/wiki/Osmose/issues#4080):
-    Details
-    Object tagged twice as node, way or relation:
-    Class 1 "Object tagged twice as node and way" 
-    Class 2 "Object tagged twice as way and relation" 
-    Class 3 "Object tagged twice as node and relation" 
+[Osmose#4080](https://wiki.openstreetmap.org/wiki/Osmose/issues#4080): \
```
Comment (Bentleysb) [inline docs/checks/duplicateMapFeatureCheck.md:5]: This \ is showing up when rendered.
Screen Shot 2021-09-07 at 8 48 44 AM

Comment (xiaohongtang2) [reply docs/checks/duplicateMapFeatureCheck.md:5]: thanks Bentley, updated it

**Review by Bentleysb — DISMISSED** [review]
LGTM. Thanks for all the testing!

**Review by rheatreena — DISMISSED** [review]

**Review by sayas01 — DISMISSED** [review]

**Comment by Bentleysb** [comment]
@xiaohongtang2, did you want to follow atiannicelli's suggestion to re-enable this check before we merge this PR?

**Comment by xiaohongtang2** [comment]
> @xiaohongtang2, did you want to follow atiannicelli's suggestion to re-enable this check before we merge this PR?

Did you want me add configuration.json in this PR to enable this check?

**Comment by Bentleysb** [comment]
@xiaohongtang2, as this seems to fix the runtime issue I think it makes sense to re-enable the check.

**Review by Bentleysb — APPROVED** [review]

**Pull request merged**